MRETestBed/Assets/TestBed Assets/Scripts/MREComponent.cs

```diff
@@ -112,7 +112,8 @@ void Start()
 		if (!_apiInitialized)
 		{
 			MREAPI.InitializeAPI(
-				DefaultPrimMaterial,
+				defaultMaterial: DefaultPrimMaterial,
+				collisionLayers: new MixedRealityExtension.Core.EngineCollisionLayers(0, 9, 10, 5),
```

Comment:
>, 9, 10, 5 [](start = 75, length = 10)

Define constants for these. Also where did these come from?

Reply:
Oh wait this is the testbed, nevermind.

---
In reply to: [335213670](https://github.com/microsoft/mixed-reality-extension-unity/pull/119#discussion_r335213670) [](ancestors = 335213670)